tests/integration/telemetry/stats/prometheus/nullvm/accesslogs_test.go

```diff
@@ -33,33 +19,51 @@ package nullvm
 
 import (
 	"fmt"
-	"strings"
 	"testing"
 	"time"
 
-	"istio.io/istio/pkg/test"
 	"istio.io/istio/pkg/test/framework"
-	"istio.io/istio/pkg/test/framework/components/echo"
-	"istio.io/istio/pkg/test/util/retry"
-	"istio.io/istio/tests/integration/security/util"
+	"istio.io/istio/pkg/test/framework/components/istio"
 	common "istio.io/istio/tests/integration/telemetry/stats/prometheus"
-	testutils "istio.io/istio/tests/util"
 )
 
 func TestAccessLogs(t *testing.T) {
 	framework.NewTest(t).
 		Features("observability.telemetry.logging").
 		Run(func(t framework.TestContext) {
 			t.NewSubTest("enabled").Run(func(t framework.TestContext) {
-				runAccessLogsTests(t, true)
+				applyTelemetryResource(t, true)
+				common.RunAccessLogsTests(t, true)
 			})
 			t.NewSubTest("disabled").Run(func(t framework.TestContext) {
-				runAccessLogsTests(t, false)
+				applyTelemetryResource(t, false)
+				common.RunAccessLogsTests(t, false)
 			})
 		})
 }
 
-func runAccessLogsTests(t framework.TestContext, expectLogs bool) {
+func TestAccessLogsDefaultProvider(t *testing.T) {
+	framework.NewTest(t).
+		Features("observability.telemetry.logging.defaultprovider").
+		Run(func(t framework.TestContext) {
+			cfg := `
+accessLogFile: ""
+extensionProviders:
+- name: file-log
+  envoyFileAccessLog:
+    path: /dev/stdout
+defaultProviders:
+  accessLogging:
+  - file-log
+`
+			ist := *(common.GetIstioInstance())
+			istio.PatchMeshConfig(t, ist.Settings().IstioNamespace, t.Clusters(), cfg)
```

Comment:
Does mesh config get written back to the pre-patched version after test finishes? Otherwise we are relying on the order of test which seems to be not ideal.

Reply:
yes, you're right, it may cause fail when another test case added, I'll fixed it.